contracts/example/tests/test.rs

```diff
@@ -26,13 +35,90 @@ fn setup_gas_service<'a>(env: &Env) -> (AxelarGasServiceClient<'a>, Address, Add
     (gas_service_client, gas_collector, gas_service_id)
 }
 
-fn setup_app<'a>(env: &Env, gateway: &Address, gas_service: &Address) -> ExampleClient<'a> {
-    let id = env.register(Example, (gateway, gas_service));
+fn setup_app<'a>(
+    env: &Env,
+    gateway: &Address,
+    gas_service: &Address,
+    interchain_token_service: &Address,
+) -> ExampleClient<'a> {
+    let id = env.register(Example, (gateway, gas_service, interchain_token_service));
     let client = ExampleClient::new(env, &id);
 
     client
 }
 
+fn setup_interchain_token_service<'a>(
```

Comment:
can you create a ticket to refactor this and expose a testutil::setup method from ITS crate? similar to what we do for the gateway